.github/renovate.json

```diff
@@ -300,6 +300,18 @@
       "addLabels": [
         "automerge"
       ]
+    },
+    {
+      "matchDatasources": [
+        "maven"
+      ],
+      "registryUrls": [
```

Comment:
I recommend migrating it in https://github.com/camunda/camunda/blob/main/bom/pom.xml#L103 (to repo.maven.apache.org/maven2) - there's no reason to use the legacy registry.

Same for https://github.com/camunda/camunda/blob/e2bd8e2c9c94f966c3ac8a7c992241442cd56bb5/tasklist/.idea/jarRepositories.xml#L7 (unless this file is dropped completely)